org.kde.peruse.json

```diff
@@ -0,0 +1,445 @@
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/gnustep/tools-make/archive/make-2_9_1.tar.gz",
+                            "sha256": "78ef0f68402c379979a9a46499ac308fe5c1512aa198138c87649ee611aedf41",
+                            "x-checker-data": {
+                                "type": "anitya",
+                                "project-id": 1220,
+                                "stable-only": true,
+                                "url-template": "https://github.com/gnustep/tools-make/archive/make-$version.tar.gz"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "name": "gnustep",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "./configure --prefix=/app",
+                        "make -j $FLATPAK_BUILDER_N_JOBS install",
+                        "ls /app/include"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/gnustep/libs-base/releases/download/base-1_29_0/gnustep-base-1.29.0.tar.gz",
+                            "sha256": "fa58eda665c3e0b9c420dc32bb3d51247a407c944d82e5eed1afe8a2b943ef37",
+                            "x-checker-data": {
+                                "type": "anitya",
+                                "project-id": 9105,
+                                "stable-only": true,
+                                "url-template": "https://github.com/gnustep/libs-base/releases/download/base-$version/gnustep-base-1.$patch.0.tar.gz"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "name": "unarchiver",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "sed -s -i 's#/usr/include/GNUstep#/app/include#g' */Makefile.linux && cd XADMaster && make -j $FLATPAK_BUILDER_N_JOBS -f Makefile.linux",
+                        "install -m755 XADMaster/unar XADMaster/lsar /app/bin/"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://cdn.theunarchiver.com/downloads/TheUnarchiverSource.zip",
+                            "sha256": "ac4cedf9b49375c0f805bda010fa03629d05ab283a0a8e35bf2dd01ae477e0a1",
+                            "strip-components": 0
+                        }
+                    ]
+                },
+                {
+                    "name": "discount",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "./configure.sh --prefix=/app --enable-all-features --with-fenced-code --shared",
```

Comment:
why not autotools?

You can do the sed with a patch as well. it's safer.

Reply:
The sed can't be done before configure and configure is not configure but ./configure.sh